gef-ui/server/controllers/amendments/cover-end-date/get-cover-end-date.test.ts

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable import/first */
```

Comment:
can this be removed please and line 5 and 6 7 and 8 imported first

Reply:
Done

Reply:
I can still see it.

Reply:
`import { getCurrencySymbol } from './getCurrencySymbol';

const getApplicationMock = jest.fn();
const getFacilityMock = jest.fn();

jest.mock('../../../services/api', () => ({
  getApplication: getApplicationMock,
  getFacility: getFacilityMock,
}));`

I did this locally in another file and it did not show any linting errors.  Not sure if it will work for you if you try again

Reply:
ignore me, it does not work

Reply:
please revert this.  We will investigate this later

Reply:
@MarRobSoftwire created a ticket in the backlog for this.